twoinputs/netdemand-solarangle-hourly.ipynb

```diff
@@ -0,0 +1,927 @@
6FMOe5/V0iNObWDLd6hD7PazNx4tlWUxPT7N3796ei3Y7MVRrPYIYoPzooyBgaXyKUsMx1CiI0+l0vWw6V5M4Hpxa9rD37UObn2/6btiuDPQrJIT4m8B/D/xd4EmaxzB9GfjExu9axE7myGSS8wsF3OAkbrVkb/n+4ZCkpUWhWhER9yiO63FhqcChic3tHx6Uo7vTlKpOUwvKRvP6bBZN9cV3K67r4jjOukStqFb8ktjAAYwE8dYjHL2kKAqVB46C9DDffrt+fVIXJI3uzmTBlqRM4QunhlAux3G4ceNGV3cYQNRqeJq+oQnkw3GDXNmun3P0wrIsZmZmSKVSeKqGsH1B3JgQ3YnWGcShy6xpGsVisWm7xu0haCPI5bkxspuJ2PoWAerBWn1c4urBgyA99MuXum5jnD+Pml3BOHsWdWGh7XpFUeql3oWqgxAQ75Cgb2oKuqpscMm0f1+pBgHuVCrgukjT7FiCb1kW8Xi863u3Ux3isBR8oCC48XHyH/8EZx58nMYiCillPXndNE1SqRS2bVMItnlj0aO6dy8g0Wc7jwfcTgz6TfNfAn9XSvnbwCst150DDm7oXkXseI5MpqjYLrPL/g9ltmxvi3Jp8HuIo5LpiIh7k5NzOaq2x0MNY+PuJg8EInWzyqallLx6JcsDu9MdR0c1BgYNiqhUkcEM0Mgh3prouo7ruui6TnV0BHd0DPPkqaZtJuOCGx1EmO1KKo5fMi2EIJdbOzZv3rzZOz1aSn/Mkb7xJdNSDl7Cq6oq8XgcaejgSWiYvdsN13WbBFmlUkEIgaZpdYc47L8Ntw/Fsbqygu1BPj1MZp1DCwYVxM70NNI0e/YRx155BS+RROoG8f/0n9qub5xDXKw5xAytrY0i3C4d0zbcIY6bzZWEdqGAoHceQSwW67mYsdO+f8JRZYMKYvB7id1MpskhBpomB4yNjVGp1ijakmFLkKtJziV2ITV9R/QRD3oUTAPtnwyfGnBnGqkidgytfcQrpdqWH7kUkrQ0bNej6qyvTDAiImL789KlZWKG2tEtvRukLZ2ZkTgn5zZHEM+tVlgqVDuWS4flk+t1iJWqL4jDdN2ddkK6E9B1HSkluq7jSUnlwQfRr11FyWbr20zGBblqe6luPtBAKR0Mw6j3Ebuuy9zcXE93GMcB6SE1fUOPjfD8olcfcStCCL+HGIkygCBuFV3lchlVVVFVlVqtVnePQ+c4rK4QQqCurOB4Em10eOAy8Xw+Tz6fR/EcMqZgodxH/CgKtQMH/D7iDiXE6vIy+qVLVE48SuXRRzDffgdlZaXlLnyH2HVdChWHpNm93Dhl6RtaTZer2PXe5BC74LfeyR6J9T2rEXboHHQhxMAl0yEJTdD68WgUxENDQ5Qdf33ooTGFhA6vLYM9s2dH9BEPehRcA451ue44cHFjdifiXmEkYTCWNDm7UMDzJLmyvW0EceN8vYiIiHsH2/V4bTbLo3uHt1TewbHpIc4vFKjYG79I99oVXwAd39MuiEN3a/0OcQXPsuopwlHK9NYjdNsMw/Bnzh59AADz1On6NrsSQR9xsfmku2D7/04aa+5orVZjZWUFx3F6lqiKQCxutEMctmRlu4xe6rgvQiAMEySIoN+3n0PcOoO40QkP+4sbBXH42VFWVnA8sMbbP2edCNOrDxw4AMCQWmMu73Tcv5oreX7O5fvXHN4cOUA1W+TSmRucz3qsVNa2t159FRSV8sMPU37sMVAE8RdebHtN6g5x1ekYqBWStvSuo65uhXzFqfcmQ+C2B6XoXiCIOxEu7nRjJ37/hJ+d9QjiuE5Ph9gwDKQRx/M80obg4TGV2bwkO7UX9eYSSnHz2nbuBIN+0/wR8LeFEO9tuEwKIY4A/wvwhUHuJEikflEI8boQ4qQQ4jdarv+nQohCw79NIcQXhRDnhBAvCCH2N1z3vweXvyOE+PGGy38iuOycEOLXB3x+EXeBI7tSnJnPk6vYSAmZ2DbpITb9H4CobDoi4t7i1FyOcs3l8f3Dd3tXmji6O43rSd65ke+/8Tp5bXaF/WMJhhPt38+e5yGEWL8grlaRllkv7duJJ6TbnfBkOhTE3tAQ9syMnzYdnGRPBr2urWXT+Zr/73RQMi2lJJ/Pc+3atXpPYjdEkGTraRscqhULHeLewVpN+yIEIthfUbP7CuLWz0GlUmkSxNVqtS6Iw+qKUCTLpRUKVpJkcrB6ac/zMAyD8fFxHnroIY4fnKLsKixmC00BXgDfu+ry8rzLm0sef6nNkLclc6+d4ysXHT7/ju2PzqrVsN58k+qRI8hkEi+VonLsIay33kTJr32vCCHqoVrFmkvC7CGIYxubt7JatpsCtaSUyIqfWC8No0m8NaJpGpqmdQ3W2okOcZjevx5BHNMEjue3PIQ9yK2LV1oig5QeKR0eHFVQBLyW2AOAObu95xEPehT8n8DbwHPA2eCyPwLeDP799wa8nyrwISnlceAR4CeEEE8BCCEeA1qXxv46sCKlPAT8Q+DvB9seBT4NPAj8BPDPhRCqEEIF/h/gJ4GjwC8F20ZsQY5MJilUnHrv23ZziAtR0nRExD1FWC59dPfWKJcOOTSRRFeVDe8jXi3ZXFgsdiyXBuolpI1BQoMgKhWk6ZcxdptFG3F3CcWoGZS2A1SPPoi6fBNtfh4AXRWMxgTzpWahkQ80ZyI4l9Z1nevXr1OtVruKlhARlB1vtEOcjukIsf6SacUKBLHTXxA3OsJhsnEngWwHY5waF5O8m8tkExmGjME+C47jEIvF6vt5ZGqYZDJJevoguq5TCYTi2RWP08seT+xS+W+PG/zakxmGDuzmI5XLfHCPiuP5vcfWqdOIapXyiRP1xyg9+QR4ktjLL9cva5ylnK/YdYOgE37JtL0uUdYL3yFuHrkkA/HvBKK3G5Zlde0j3qmCOB6Pr2skXiL4aBYdv3rBMNoXQd1gLFNCh7guODKs8IIYwzVMzG0erDXoHOIy8DTwq8DzwDeBl4C/AXxUSjn
```

Comment:
Outputs work again after this point.

Reply:
All of the outputs should be there except for the cell I interrupted.